src/frontend/use-config.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactElement, useContext, createContext } from 'react';
```

Comment:
Can you exclude this file from the API Docs https://github.com/auth0/nextjs-auth0/blob/beta/typedoc.js#L4

Reply:
Sure 👍🏼